tests/compiler/std/symbol.optimized.wat

```diff
@@ -1490,6 +1489,9 @@
   call $~lib/string/String#concat
  )
  (func $~lib/symbol/_Symbol#toString (; 28 ;) (type $FUNCSIG$ii) (param $0 i32) (result i32)
+  (local $1 i32)
+  i32.const 296
+  local.set $1
```

Comment:
I'm wondering why this happened? `$1` not using inside this function at all

Reply:
Hmm, maybe that's retained and released in the function, with post-assemblyscript removing those, leaving the set behind since no further passes are run after that. Strange that it exists in the first place, though.